### ISSUE TITLE
Le tableau des scores ne fonctionne pas 🥇

### ISSUE THREAD

**Issue by AdrKacz** [kiwily/game-jam-merge#4]
**Describe the bug**
Impossible de charger les scores (donc impossible de voir si le score a été enregistré non plus)

**To Reproduce**
1. Ouvrir l'app
2. Faire une partie
3. Cliquer sur "Submit"
4. Ça tourne dans le vide ...

1. Ouvrir l'app
2. Cliquer sur "Scores"
3. Ça tourne dans le vide ...

**Expected behavior**
Pouvoir envoyer son meilleur score et voir les scores des autres🥈 

**Screenshots**
![image](https://user-images.githubusercontent.com/61620013/148644738-53fc4d24-df34-41fa-ae00-662220c50bc9.png)
![image](https://user-images.githubusercontent.com/61620013/148644745-b8a01740-b6c0-4905-adc5-7c72f5dfe967.png)

**Smartphone (please complete the following information):**
 - Device: iPhone 13 mini
 - OS: iOS 15.1.1

**Comment by kiwily** [comment]
normal faut avoir un server qui tourne derrière!

**Comment by AdrKacz** [comment]
Yes je sais, j'ai viré le serveur, fausse manip' 😅 Je le note juste là histoire de pas oublier dans 2 semaines 😌
Je me disais, il faudrait peut être mettre le code du serveur dans le même repo 🧐
On peut PUSH seulement un folder sur Heroku si le serveur n'est pas à la racine 📁
